### DISCUSSION TITLE
Upiększenie userMessage dla "Parametr o tej nazwie już istnieje"

### DISCUSSION THREAD

**Discussion by Radeq** [allegro/allegro-api#5047] [Suggestions - rejected]
Witam,

Proszę poprawę komunikatu userMessage przy błędzie dodania własnego parametru dodatkowego który ma taką samą nazwę jak parametr kategorii. 

Przypadek: Klient ma na ofercie ustawiony własny parametr - kolor zielony. Po jakimś czasie Allegro aktualizuje/zmienia kategorię i parametr kolor został dodany do cech. Przy próbie edycji wyskakuje błąd:

OBECNIE
```
0 => {#6610 ▼
      +"code": "BAD_REQUEST"
      +"message": "Parameter already exists: name='Kolor', id='127448'"
      +"details": null
      +"path": "$.parameters[0].name"
      +"userMessage": "Parametr o tej nazwie już istnieje"
    }
```

Użytkownikowi wyświetlam userMessage który mało wyjaśnia.

Proponuję by userMessage zawierał nazwę parametru, np:

PROPOZYCJA
```
0 => {#6610 ▼
      +"code": "BAD_REQUEST"
      +"message": "Parameter already exists: name='Kolor', id='127448'"
      +"details": null
      +"path": "$.parameters[0].name"
      +"userMessage": "Nie możesz dodać parametru dodatkowego o takiej samej nazwie jak parametr: kolor"
    }
```

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za zgłoszenie, przekazuję tę uwagę już bezpośrednio do odpowiedniego zespołu.

**Comment by PrzemyslawLukanowski** [comment]
Witam,

Zamykam sugestię ze względu [brak dalszego wsparcia dla własnych parametrów](https://developer.allegro.pl/news/parametry-wlasne-zakonczymy-wsparcie-dla-funkcjonalnosci-jnR1BD4K7uV).
